cibyl/cli/main.py

```diff
@@ -18,6 +18,12 @@
 from cibyl.orchestrator import Orchestrator
 from cibyl.utils.logger import configure_logging
 
+# CLASSMAP is used to map the plugins classes
+# with plugin name so it should be easy to
+# import while running.
+
+CLASSMAP = {"openstack": "Deployment"}
```

Comment:
Where is it used?

Reply:
It is not used anywhere for now. I'll remove it for now.

Use case which i was trying to load specific classes once we loaded the plugins.